desktop/libs/kafka/src/kafka/kafka_api.py

```diff
@@ -20,7 +20,7 @@
 import logging
 import sys
 
-from desktop.api_public import _get_interpreter_from_dialect
+from desktop.api_public import *
```

Comment:
Please avoid * and put the explicit names

Reply:
but here if i put the explicit name then I am getting this error `ImportError: cannot import name '_get_interpreter_from_dialect' from partially initialized module 'desktop.api_public' (most likely due to a circular import)`